src/main/java/com/minecolonies/colony/buildings/BuildingFarmer.java

```diff
@@ -100,35 +410,58 @@ public View(ColonyView c, BlockPos l)
         @Override
         public void deserialize(ByteBuf buf)
         {
+            fields = new ArrayList<>();
             super.deserialize(buf);
+            assignFieldManually = buf.readBoolean();
+            int size = buf.readInt();
+            for(int i = 1; i <= size; i++)
+            {
+                FieldView fieldView = new FieldView();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'fieldView' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)